package.json

```diff
@@ -48,5 +53,14 @@
     "react-dom": "^16.13.1",
     "scheduler": "^0.19.1",
     "wp": "0.0.3"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```

Comment:
because we primarly work with SASS files, we should includem them as well @elsurn (and mabye exclude css)

Reply:
@col-tom good idea! We could use also https://stylelint.io/ . What do you think?